source/SIL.AppBuilder.Portal.Frontend/src/ui/components/semantic-extensions/responsive-sub-navigation/index.tsx

```diff
@@ -15,7 +15,7 @@ class ResponsiveSubNav extends React.Component<IProps & RouteComponentProps<{}>>
     const { items, location } = this.props;
 
     const matchingItem = items.find(i => {
-      return (i.to === location.pathname);
+      return (location.pathname.includes(i.to));
```

Comment:
this should be a regex matching the beginning yeah? otherwise any substring will match.

Reply:
updated